common/lib/xmodule/xmodule/js/fixtures/tabs-edit.html

```diff
@@ -41,10 +41,10 @@ <h2 class="title modal-window-title">Mock Modal Title</h2>
             <div class="modal-actions">
                 <ul>
                     <li class="action-item">
-                        <a href="#" class="button action-primary action-save">Save</a>
+                        <button type="button" class="button action-primary action-save">Save</button>
```

Comment:
Not if they're in CMS, which we're not focusing on in this PR. The work here should mostly be contained/scoped to the LMS.